### PR TITLE
provider/azurerm: support import of routes, fix route_table

### PR DESCRIPTION
**Pull request by pmcatominey**
Added import functionality to:

* `azurerm_route`
* `azurerm_route_table`

Fix `azurerm_route_table` failing to actually set any routes declared within.

### CODE REVIEW

File: builtin/providers/azurerm/resource_arm_route_table.go [hunk 1]
```diff
@@ -73,7 +76,6 @@ func resourceArmRouteTable() *schema.Resource {
 
 			"subnets": {
 				Type:     schema.TypeSet,
-				Optional: true,
```
Comment (stack72) [inline builtin/providers/azurerm/resource_arm_route_table.go]: Is this a breaking change?

**Comment by stack72** [comment]
Just got 1 question - but apart from that, LGTM!

File: builtin/providers/azurerm/resource_arm_route_table.go (hunk 1 above)
Comment (pmcatominey) [inline builtin/providers/azurerm/resource_arm_route_table.go]: It shouldn't be, that value can't actually be set, the code was never writing the value to the API either.

Comment (stack72) [inline builtin/providers/azurerm/resource_arm_route_table.go]: Rgr! Thanks :)

**Comment by stack72** [comment]
Thanks for the response - this LGTM! Thanks :)

**Pull request merged**